src/services/transfers.service.ts

```diff
@@ -69,11 +71,21 @@ class TransfersService {
   public async findTransferByFilterParams(args: TransfersByCursorOptions): Promise<Transfer[]> {
     const { page, limit, status, sender } = args
 
-    const queryParams = this.prepareQueryParams({ page, limit, status, sender })
+    const queryParams = this.prepareQueryParams({ page, limit, status })
     const { skip, take, where } = queryParams
 
-    const transfer = await this.transfers.findMany({
-      where,
+    let whereToUse = { ...where } as { [key: string]: string | { [key: string]: string } | undefined }
```

Comment:
What's this?
This function is now becoming unruly.
I don't think you necessarily need to add this.
For filtering by address create another function.

Reply:
when I added the changes to the schema, something changed at the type level, and this was the only way I could make the linter silent. Since `where` could be `status = pending` or with this addition, `{ account: 0x0... }`, an object.

Reply:
Try to figure out what's wrong.
We shouldn't need this whereToUse at all.